external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/EventHubSpout.java

```diff
@@ -152,7 +152,7 @@ public void open(Map config, TopologyContext context, SpoutOutputCollector colle
     try {
       preparePartitions(config, totalTasks, taskIndex, collector);
     } catch (Exception e) {
-      logger.error(e.getMessage());
+	  collector.reportError(e);
```

Comment:
Someone else can correct me here, but I don't think collector.reportError puts anything in the log, so it might be nice to have both this and logger.error